src/Raven.Server/Documents/ETL/EtlLoader.cs

```diff
@@ -689,14 +719,36 @@ public void HandleDatabaseRecordChange(DatabaseRecord record)
                                 myElasticSearchEtl.Remove(existing);
                             }
 
+                            break;
+                        }
+                    
+                    case SnowflakeEtl elasticSearchEtl:
```

Comment:
```suggestion
                    case SnowflakeEtl snowflakeEtl:
```

and fix the usages below (you compare against `elasticSearchEtl`)

Reply:
done